app/scraper/spiders/feed/financiero.py

```diff
@@ -0,0 +1,54 @@
+import scrapy
+from datetime import datetime
+from utils.text import remove_blank_lines
+from utils.text import normalize_titles
+from scraper.items import Feed
+
+
+class ElFinancieroFeedSpider(scrapy.Spider):
+    name = 'ElFinanciero:Feed'
+    allowed_domains = [
+        'elfinanciero.com.mx'
+    ]
+    start_urls = [
+        'https://www.elfinanciero.com.mx/bloomberg-businessweek',
+        'https://www.elfinanciero.com.mx/economia',
+        'https://www.elfinanciero.com.mx/empresas',
+        'https://www.elfinanciero.com.mx/mercados',
+        'https://www.elfinanciero.com.mx/nacional',
+        'https://www.elfinanciero.com.mx/tech'
```

Comment:
Please remove all this URL, use arguments to run the spiders in a specific tag, please all these tags put them in the readme.md you can make it as el universal news.